Move secret examples to own folder (#203)
SAP/stewardci-core: docs/examples/secrets/README.md

# Secrets examples

To run these examples you need to [create a Tenant](../README.md#tenant) and [set the environment variable](../README.md#pipelinerun) `TENANT_NAMESPACE` accordingly.

## Basic examples

You can find some basic examples in [secretExamples.yaml](secretExamples.yaml).

## Missing secrets

> If a secret is missing the pipelien run will fail with an content error.

Reviewer: ```suggestion
If a secret is referenced in a pipeline run spec but does not exist, the pipeline run will fail with result `error_content`.
```

```bash

> spr=$(kubectl -n "$TENANT_NAMESPACE" create -f pipelinerun_missing_secret.yaml -oname)
> kubectl  -n "$TENANT_NAMESPACE" get $spr -owide

Reviewer: ```suggestion
spr=$(kubectl -n "$TENANT_NAMESPACE" create -f pipelinerun_missing_secret.yaml -o name)
kubectl  -n "$TENANT_NAMESPACE" get "$spr" -o wide
```

```

```
NAME                  STARTED   FINISHED   STATUS     RESULT          MESSAGE
missingsecret-rsww9   12s                  finished   error_content   ERROR: preparing failed ...
```

## Secret renaming

> It is possible to [rename secrets](../secrets/Secrets.md). This can be tested with the pipeline run in the file [pipelinerun_secret_rename.yaml](pipelinerun_secret_rename.yaml). As preparation you need to create the secret with the rename annotation. You can find the renamed secret in the run namespace as listed below.

Reviewer: ```suggestion
It is possible to [rename secrets](../secrets/Secrets.md).
This can be tested with the pipeline run in file [`pipelinerun_secret_rename.yaml`](pipelinerun_secret_rename.yaml).
As a preparation you need to create the secret with the rename annotation.
You can find the renamed secret in the run namespace as listed below.
```

```bash

> kubectl -n "$TENANT_NAMESPACE" create -f secret_rename.yaml
> spr=$(kubectl -n "$TENANT_NAMESPACE" create -f pipelinerun_secret_rename.yaml -oname)
> runnamespace=$(kubectl  -n "$TENANT_NAMESPACE" get $spr -ojsonpath='{.status.namespace}')
> kubectl -n "$runnamespace" get secret

Reviewer: ```suggestion
kubectl -n "$TENANT_NAMESPACE" create -f secret_rename.yaml
spr=$(kubectl -n "$TENANT_NAMESPACE" create -f pipelinerun_secret_rename.yaml -o name)
runnamespace=$(kubectl  -n "$TENANT_NAMESPACE" get "$spr" -o jsonpath='{.status.namespace}')
kubectl -n "$runnamespace" get secret
```

# Secrets examples

> To run this examples you need to [create a Tenant](../README.md#tenant) and [set the environment variable](../README.md#pipelinerun) `TENANT_NAMESPACE` accordingly.

Reviewer: ```suggestion
To run these examples you need to [create a Tenant](../README.md#tenant) and [set the environment variable](../README.md#pipelinerun) `TENANT_NAMESPACE` accordingly.
```

## Missing secret

> 

Reviewer: It looks like some documentation should go here.

Author: Fixed
